src/http/http_consts.h

```diff
@@ -11,7 +11,9 @@ namespace http
     static constexpr auto AUTHORIZATION = "authorization";
     static constexpr auto CONTENT_LENGTH = "content-length";
     static constexpr auto CONTENT_TYPE = "content-type";
+    static constexpr auto DATE = "date";
```

Comment:
Do we still want the date in here? I thought the latest spec didn't require it and so we wanted to remove it.

Reply:
We're not enforcing that this is signed, just naming the standard header for use in the test, and potentially for future use when building HTTP requests.